docs/guide-release-process.md

```diff
@@ -268,12 +269,18 @@ npx @rnx-kit/rn-changelog-generator \
 ```
 </details>
 
-### Step 8: Create the GitHub Release
+### Step 8: Publish the GitHub Release
 
 > [!Note]
 > For large releases with many changelog changes such as RC0 (e.g. `0.81.0-rc.0`) or Major release (e.g. `0.81.0`) the link to CHANGELOG.md at the bottom of the Github release is enough.
 
-For React Native (**`>= 0.80`**), a draft Github release is generated by the job **Publish Release**.
+For React Native (**`>= 0.80`**), a draft Github release is generated by the job [**Publish Release**](https://github.com/facebook/react-native/actions/workflows/publish-release.yml). Verify all details and links are valid and publish it.
+
+#### Verify DevTools Github Release Assets
+The Publish Release job uploads several DevTools assets to GitHub Release Assets.
+
+Once the release is published, [Validate DotSlash Artifacts](https://github.com/facebook/react-native/actions/workflows/validate-dotslash-artifacts.yml) will validate that these assets were uploaded correctly. Ensure that this action run is green and that the assets exist on the release:
```

Comment:
> Ensure that this action run is green and that the assets exist on the release:

@motiz88 @vzaidman  Can we get this job to open a workplace post or a task or a discord message rather than having the release crew to explicitely check it?

Reply:
The only way for the release to be "green" with missing assets is if it was created manually, effectively skipping the automated draft release job. If that job executes and completes successfully, we have a valid release with valid assets. The post-release validation workflow exists only for extra safety (basically to catch deviations from our routine, automated process).

Reply:
@motiz88 say the release job has indeed failed for an unrelated reason (like was the case in this release), and a manual release was created instead following the wiki, would the verification job still be red?

What do you think we should change in the wiki in this case? Currently it just tells users to create the release manually.

Reply:
I think we need to change that so releases get created either with a script locally, or with the same script as part of a workflow - never manually on GitHub.

Reply:
The verification job would be red on any release performed incompletely, yeah.